README.md

```diff
@@ -130,28 +130,61 @@ Looking towards the future, our vision is to enhance this platform by integratin
 
 ### Installation
 
-_TODO
-_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
-
-1. Solder your PARTS CRP Board
-2. Solder accessories like the IMU
-3. Assemble the Pololu Romi Chassis
-   ```sh
-   npm install
-   ```
-4. Enter your API in `config.js`
-   ```js
-   const API_KEY = 'ENTER YOUR API';
-   ```
+Welcome to the setup guide for the PARTS Common Robotics Platform. Follow these steps to prepare your hardware and software for an exciting journey into robotics!
+
+#### 1. Solder Your PARTS CRP Board
+Soldering the PARTS CRP board is the first crucial step. Follow the instructions carefully to ensure a successful setup.
+
+- **Interactive BOM**: Utilize the [Interactive BOM](https://htmlpreview.github.io/?https://github.com/portlandrobotics/common_platform/blob/master/hardware/romi_board/bom/ibom.html) for an easier soldering process.
+- **Headers for Connectivity**: We recommend using headers for connections between boards. This makes disassembly possible if needed.
+  - Use female (socket) headers on the bottom board.
+  - Use male (pin) headers on the top board. These are typically included with all breakout boards.
+- **MOSFET Orientation**: Ensure the correct orientation of the MOSFETs during soldering.
+- **Resistor for Battery Voltage Divider**: Choose the resistor based on your battery choice:
+  - Use a 2M ohm resistor (R1) for 6 AA batteries.
```

Comment:
A quick explanation of why these resistors make a difference. Show the work (Math)